### ISSUE TITLE
When trying to add service data to 2nd notification the data gets removed after saving

### ISSUE THREAD

**Issue by bobvmierlo** [nielsfaber/alarmo#107]
Hi Niels,

I've just tried to add a HTML5 notification to my existing iOS notification.
However when adding the servicedata in the yaml editor in the same way you add extra data to the iOS notification the extra servicedata get's deleted again after clicking the save button.

This is the yaml I see when opening the editor:
```
name: Alarm Away Notify
modes:
  - armed_away
triggers:
  - state: triggered
actions:
  - service: notify.mobile_app_iphone_11_van_bob
    service_data:
      message: 'The alarm is triggered! Cause: {{open_sensors}}.'
      title: Alarm!
      data: &ref_0
        attachment:
          content-type: jpeg
        entity_id: camera.mycamera
        push:
          category: CAMERA
          sound:
            critical: 1
            name: US-EN-Alexa-Motion-Detected-Generic.wav
            volume: 1
  - service: notify.html5
    service_data:
      message: 'The alarm is triggered! Cause: {{open_sensors}}.'
      title: Alarm!
      data: *ref_0
area: '1611601388'
```

Then I change it to this **(I purposely left the '&ref_0 and '*ref_0' parts in as I guessed these are related to your code)**:
```
name: Alarm Away Notify
modes:
  - armed_away
triggers:
  - state: triggered
actions:
  - service: notify.mobile_app_iphone_11_van_bob
    service_data:
      message: 'The alarm is triggered! Cause: {{open_sensors}}.'
      title: Alarm!
      data: &ref_0
        attachment:
          content-type: jpeg
        entity_id: camera.mycamera
        push:
          category: CAMERA
          sound:
            critical: 1
            name: US-EN-Alexa-Motion-Detected-Generic.wav
            volume: 1
  - service: notify.html5
    service_data:
      message: 'The alarm is triggered! Cause: {{open_sensors}}.'
      title: Alarm!
      data: *ref_0
        actions:
          - action: sound_alarm
            icon: "/static/icons/favicon-192x192.png"
            title: Sound alarm!
area: '1611601388'
```

Then after saving the above and reopening the editor I see this first part of code again with this part missing:
```
data:
  actions:
    - action: sound_alarm
      icon: "/static/icons/favicon-192x192.png"
      title: Sound alarm!
```

p.s.: when I remove the '&ref_0 and '*ref_0' parts the code gets removed aswell, also these parts of code get placed back automatically aswell.

p.p.s. love the new 1.5.0 layout and the area addition!

**Comment by nielsfaber** [comment]
I must say I never tested this scenario, I never expected the actions to get mixed up.
I will try if I can reproduce it, because what you’re trying should just work.
The ref_0 pointers are not created by Alarmo anywhere. But they are allowed in yaml: https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

**Comment by nielsfaber** [comment]
This problem should be solved with https://github.com/nielsfaber/alarmo/commit/b550f8df89a59ca2fb88b9e056fbcf8a70780045
It has to do with the fact that the UI editor only supports a single notification (which can be sent to multiple targets).
When you save the notification with 2 different notifications, it is stored correctly.
But as you edit this notification again (or even switch from yaml -> UI editor) all notifications would get overwritten by the first one.
The bottom line is: if you want to create multiple different notifications, stick to the yaml editor. 
Your data should remain intact with the fix.

**Comment by nielsfaber** [comment]
v1.5.2 is released which contains this fix.
Please test it and close the issue if the problem is resolved.

**Comment by bobvmierlo** [comment]
Works perfectly! Thanks
